exchanges/margin/margin.go

```diff
@@ -32,6 +32,10 @@ func (t Type) String() string {
 		return isolatedStr
 	case Multi:
 		return multiStr
+	case SpotIsolated:
+		return spotIsolatedStr
+	case Cash:
+		return cashStr
```

Comment:
Missing test coverage and could you please make sure the zero return is covered as well?